dist/bash-completion/tpm2-tools/tpm2_completion.bash

```diff
@@ -0,0 +1,153 @@
+#!/bin/sh
+# bash completion for tmp2-tools                                 -*- shell-script -*-
+
+
+_tpm2_tools()
+{
+
+
+
+    local cur prev words cword
+    _init_completion || return
+    local common_options=(-h --help -v --version -V --verbose -Q --quiet -Z --enable-errata -T --tcti=)
+    local aux1=$( ${COMP_WORDS[0]} -h summary 2>/dev/null )
+    local aux2=$( echo "${aux1}" | tr "[]|" " " | awk '{if(NR>2)print}' | tr " " "\n")
+    suggestions=("${aux2[@]}" "${common_options[@]}") #generate all the opts for the tool
+    local halg=(0x4 sha1 0xB sha256 0xC sha384 0xD sha512 0x12 sm3_256)
```

Comment:
I wonder if want to list the hex values, probably the pretty names are enough. What do you think @williamcroberts ?